lib/src/main/kotlin/com/swmansion/starknet/data/types/Contracts.kt

```diff
@@ -48,9 +48,20 @@ data class FunctionAbiEntry(
     val name: String,
     val inputs: List<AbiEntry>,
     val outputs: List<AbiEntry>,
-    val type: AbiEntryType,
     val stateMutability: StateMutabilityType? = null,
-) : AbiElement()
+) : AbiElement() {
+    val type: AbiEntryType = AbiEntryType.FUNCTION
+}
+
+@SerialName("constructor")
+@Serializable
+data class ConstructorAbiEntry(
```

Comment:
Could you explain why this is included in this PR?

Reply:
In function `StandardAccount.ensureCairoVersion()` which determines Cairo version, there is a line 
```kotlin
val contract = provider.getClassAt(address).send()
``` 
It gives us a contract class definition, including its abi. Running some tests results in getting an abi containing entries with serial name `constructor`. It looks like such case didn't occur in any of our tests before, so it wasn't covered and adding `ConstructorAbiEntry` was omitted. Here is [CI with failure of mentioned tests](https://github.com/software-mansion/starknet-jvm/actions/runs/10005252217/job/27655577538).